src/conn/conn_layered.c

```diff
@@ -48,8 +48,9 @@ __layered_get_disagg_checkpoint(WT_SESSION_IMPL *session, const char **cfg,
     if (page_log->pl_get_complete_checkpoint_ext == NULL)
         WT_ERR(ENOTSUP);
 
-    WT_ERR(page_log->pl_get_complete_checkpoint_ext(page_log, &session->iface,
-      complete_checkpoint_lsn, NULL, complete_checkpoint_timestamp, complete_checkpoint_metadata));
+    ret = page_log->pl_get_complete_checkpoint_ext(page_log, &session->iface,
+      complete_checkpoint_lsn, NULL, complete_checkpoint_timestamp, complete_checkpoint_metadata);
+    WT_ERR_NOTFOUND_OK(ret, true);
```

Comment:
Return code is handled on the line https://github.com/wiredtiger/wiredtiger/blob/ddffa5daab1ba2575cfdbb14c6c10ba13924c25a/src/conn/conn_layered.c#L1244

Reply:
I mean here the code can be simplified to just using WT_ERR. This change is redundant.